specification/quantum/data-plane/Microsoft.Quantum/preview/2019-11-04-preview/quantum.json

```diff
@@ -0,0 +1,552 @@
+            "schema": {
+              "$ref": "#/definitions/restError"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "Jobs_Put",
+        "description": "Create a job.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/jobIdParameter"
+          },
+          {
+            "name": "jobDefinition",
+            "description": "The complete metadata of the job to submit.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/jobDetails"
+            }
+          }
+        ],
+        "x-ms-examples": {
+          "Submit a new job for execution to a Quantum Workspace": {
+            "$ref": "./examples/jobPut.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/jobDetails"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/jobDetails"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/restError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "Jobs_Delete",
+        "description": "Cancel a job.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/jobIdParameter"
+          }
+        ],
+        "x-ms-examples": {
+          "Cancel the execution of an existing job": {
+            "$ref": "./examples/jobDelete.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/jobDetails"
+            }
+          },
+          "204": {
+            "description": "NoContent - The job was successfully cancelled."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/restError"
+            }
+          }
+        }
+      }
+    },
+    "/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Quantum/workspaces/{workspaceName}/providerStatus": {
```

Comment:
Does it make sense to use `/providers` as the last segment? Having a GET operation that returns a list of things where the last segment is in singular is a bit odd.

Reply:
We talked about this during the review. We do plan to rename it but it will be in vnext of the REST API (along with a bunch of other changes).